scripts/phase8/run_phase8_closure_pack.py

```
@@ -35,10 +36,106 @@ class _Phase8Config:
+    )
+    attribution = (
+        cfg.repo_root / "api-contracts" / "dist" / "openapi" / "v1" / "attribution.bundled.yaml"
+    )
+    if not reconciliation.exists() or not attribution.exists():
+        missing: list[str] = []
+        if not reconciliation.exists():
+            missing.append(str(reconciliation))
+        if not attribution.exists():
+            missing.append(str(attribution))
+        raise RuntimeError(f"Canonical OpenAPI bundle(s) missing: {', '.join(missing)}")
+    joined = f"{_sha256_file(reconciliation)}|{_sha256_file(attribution)}".encode("utf-8")
```

Ruff (0.15.0)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)